### ISSUE TITLE
sqlite3 - deleting a column does not delete the column.

### ISSUE THREAD

**Issue by anothermindbomb** [dbeaver/dbeaver#11100]
#### System information: 
- Windows 10
- Dbeaver 7.3.3
- No Additional Extensions

#### Connection specification:
- sqlite3
- sqlite3
- No proxies/tunnels

#### Describe the problem you're observing:

When removing a column for an existing table, SQL is generated to create a temp table & copy the contents into it, drop the original table, re-create it and copy the contents from temp back into it.

Unfortunately, the re-creation of the table you're dropping the column from re-introduces the column you're trying to remove.

#### Steps to reproduce, if exist:

`create table A (col1 TEXT, col2 TEXT)
`
Now delete col1 by right-clicking it in the database navigator panel, selecting "Delete" and view the generated SQL...

```
-- Drop column col1

CREATE TEMPORARY TABLE temp AS
SELECT
  col2
FROM A;

DROP TABLE A;

CREATE TABLE A (
	col1 TEXT,
	col2 TEXT
);

INSERT INTO A
 (col2)
SELECT
  col2
FROM temp;

DROP TABLE temp;


```
As you can see, table A is dropped and re-created with Col1 back in there, but now empty.

**Comment by ShadelessFox** [comment]
Hello, @anothermindbomb,

Unfortunately, I cannot reproduce your issue. After following steps you've described, the following SQL script was generated:

```
-- Drop column col1

CREATE TEMPORARY TABLE temp AS
SELECT
  col2
FROM A;

DROP TABLE A;

CREATE TABLE A (
	col2 TEXT
);

INSERT INTO A
 (col2)
SELECT
  col2
FROM temp;

DROP TABLE temp;
```
As you can see, only `col2` is included to new table.

I doubt the reason hides underneath version you're currently using, but don't you mind upgrading to newer version to see if the problem still occurs?

**Comment by anothermindbomb** [comment]
I was concerned I'd made an arse of myself and done something stupid, but I've replicated this on the early access version 7.3.4.

I created my table A with 2 columns, expanded the table in the navigator and right clicked on Col1 and selected delete. The following screenshot shows I'm using 7.3.4 with the generated script to create a temp table to save the contents of table A, drops table A, re-creates table A with Col1 _still_ in there and then copies the temp table back into it leaving me with Table A, 2 columns and the contents of Col2.

Normally I loath pasting screenshots, but in this case I'm trying to convince myself what I'm describing is accurate and it seems to be:

![image](https://user-images.githubusercontent.com/1065272/105617438-37791a00-5dd6-11eb-9b31-13ec777f7815.png)
